### ISSUE TITLE
Patternfly Alignment issue

### ISSUE THREAD

**Issue by SachinRammoorthy** [jboss-outreach/certifier#54]
As can be seen in #53 , there is some alignment issue with Patternfly. Have already verified with others, and this problem does not just exist for me.

**Comment by zwliew** [comment]
The alignment issue can be fixed with some CSS, although it does seem rather hacky.

**Comment by divyanshkul** [comment]
@zwliew I think so it was caused by the extra bootstrap.min.css ```(lib/bootstrap/css/bootstrap.min.css)``` in the imports as can be seen [here](https://github.com/jboss-outreach/certifier/pull/55/files#diff-82794e294ff5fbfcff68cfa6b753fea4L10)

**Comment by zwliew** [comment]
Yeah, importing our own copy of Bootstrap conflicts with Patternfly's styles.

**Comment by zwliew** [comment]
Removed the Bootstrap styles here: https://github.com/jboss-outreach/certifier/pull/71
